components/style/Font.js

```diff
@@ -121,6 +121,17 @@ export default () => (
         unicode-range: U + 0000-00ff, U + 0131, U + 0152-0153, U + 02bb-02bc, U + 02c6, U + 02da,
           U + 02dc, U + 2000-206f, U + 2074, U + 20ac, U + 2122, U + 2212, U + 2215;
       }
+      
+      /* latin */
+      @font-face {
+        font-family: 'Space Mono';
+        font-style: normal;
+        font-weight: 400;
```

Comment:
Please add `font-display: swap;` following the other font faces.

Reply:
Should I add `font-display: swap;` to every font faces?

Reply:
Ohhhh whoops! Never mind (: that change is on another PR (#458)